### ISSUE TITLE
Profiles or the ability to add chained autoscale axis

### ISSUE THREAD

**Issue by SilMon** [mgth/LittleBigMouse#120]
The program has currently no support for profiles, which makes it especially annoying if one switches on one display between horizontal and vertical layout. The solution for this problem would either be to include setup profiles or to add the ability chain axis so they will be automatically scaled to have the same size.

**Comment by alexandrebastien** [comment]
Yeah, +1. Profile would be really useful if you have a different setup at home and at work.
